### ISSUE TITLE
Dear author, i am a Chinese student.

### ISSUE THREAD

**Issue by Crownwzt** [yitong91/StoryGAN#3]
I am attracted to your article, but i am confused in the implementation. I have generated CLEVR_single_obj_dict_rgb.npy and whole dataset, but the code clevr_data.py need to be revised. I have changed the self.srt and self.edn, but Running tips ValueError: operands could not be broadcast together with shapes (0,) (1,8). Can you tell me how to revise in detail. Thank you so much

**Comment by Crownwzt** [comment]
I have found the reason why the error may be exist in the whole dataset. The  dimension feature vector in the small dataset  is 75, while the dimension feature vector exacted from description_generation in 16. 
Can you tell me how to solve the problem? Thank you so much.

**Comment by yitong91** [comment]
The 75 dimensional feature is generated from line 61-68 in [vec_generator.py]( https://github.com/CrickWu/Clevr-for-StoryGAN/blob/master/description_generation/vec_generator.py). In order to use this feature, uncomment L61-L68 and comment L70-L80.

**Comment by Crownwzt** [comment]
Thank you for your contribution, but there may be existing some errors. The running tips img_feature[idx*18:(idx+1)*18] = basic_feature(ground_truth, idx)
ValueError: cannot copy sequence with size 13 to array axis with dimension 18

**Comment by Crownwzt** [comment]
In your recent contribution--the vec_generation, your want to change feature dimensions exacted, but the return feature list is a 13 dimensions vector not an 18-dimensional vector, how to assign img_feature?

**Comment by yitong91** [comment]
Thanks for you interest. I'll ask my co-author and fix it.

**Comment by Crownwzt** [comment]
Thank you so much, may i have your wechat number?

**Comment by Crownwzt** [comment]
Dear author can you tell me how to fix it?

**Comment by yitong91** [comment]
Is the bug fixed? The dataset is updated.

**Comment by Crownwzt** [comment]
The bug has been fixed，but in the performance Loss_D: 0.0941 Loss_G: 2.3274
                     Loss_real: 0.0890 Loss_wrong:0.0001 Loss_fake 0.0102
                     accG: 0.4531 accD: 0.9844
                     Total Time: 99.64sec
why accG: 0.4531 accD: 0.9844 ？ Are the two labels reversed？

**Comment by Crownwzt** [comment]
A epoch and  the performance may correspond to a batch=64 images, in your code?

**Comment by Crownwzt** [comment]
In my implementation, the performance seems like over-fitting in the test samples. Is that due to the size of clevr-dataset (800 training 100 testing), which is generated by the provided code? Can you tell me how much images may be suitable?

**Comment by yitong91** [comment]
Do you mean over-fit to the training samples? Actually, we use 10,000 stories samples for training and 3000 for testing. 800 training samples are still not enough. And you also need to change the srt and edn index in clevr_dataset.py in order to read the whole dataset.

**Comment by Crownwzt** [comment]
In the performance Loss_D: 0.0941 Loss_G: 2.3274
Loss_real: 0.0890 Loss_wrong:0.0001 Loss_fake 0.0102
accG: 0.4531 accD: 0.9844
Total Time: 99.64sec
why accG: 0.4531 accD: 0.9844 ？ Are the two labels reversed？

**Comment by Crownwzt** [comment]
If i use 10,000 stories samples for training and 3000 for testing, the hyperparameter should be changed? Can you provide the hyperparameter in your paper toward the whole dataset?

**Comment by yitong91** [comment]
accG means the label prediction accuracy when updating the generator. accD is the accuracy of label prediction when updating the discriminator. It is reasonable because we share some layers of the discriminator and the label classifier. Updating the discriminator can improve the prediction accuracy.

**Comment by yitong91** [comment]
I remember the hyper parameters of the code is set for the whole dataset.

**Comment by Crownwzt** [comment]
clevr.yml is not only suitable for the part samples but also for the whole dataset?

**Comment by yitong91** [comment]
Yes. clevr.yml suits for all dataset. Training only on part samples cannot get reasonable results actually. It is just for illustration of the data structure.
